templates/index.html.twig

```diff
@@ -51,7 +51,7 @@
 <div id="userlinks" style="margin-top: 30px;">
   <ul>
 
-    <li><a href="/language">Languages</a></li>
+    <li><a href="/language/index">Languages</a></li>
```

Comment:
Hm, when I go to `http://localhost:8080/language/index`, I get the following: `No route found for "GET http://localhost:8080/language/index": Method Not Allowed (Allow: POST)`.

`/language/index` doesn't actually exist ... `/language` is a route mapping in `src/Controller/LanguageController.php`:

```
    #[Route('/', name: 'app_language_index', methods: ['GET'])]
    public function index(LanguageRepository $languageRepository): Response
```

Does that change actually work on your machine?

Reply:
It's does work on my machine (Ubuntu 22). I just used the same pattern as the rest of the code (e. g. ``href="/text/index"`` two lines below). I need to learn more symfony magic I guess :thinking:

Reply:
Interesting, maybe there's an Apache rewrite happening or something.  Symfony's front controller should have throw an exception (that's where it uses the routes).  But if the line `<li><a href="/language">Languages</a></li>` is raising a warning for you, then you could leave this change in, and change the route in the src/Controller/LanguageController.cs.